scripts/ci/Dockerfile

```diff
@@ -112,34 +112,41 @@ USER runner
 
 CMD [ "jest", "main.spec.js" ]
 
-FROM mcr.microsoft.com/playwright:v1.31.2-focal as playwright-base
+
+
+ARG PLAYWRIGHT_VERSION
+FROM mcr.microsoft.com/playwright:v${PLAYWRIGHT_VERSION}-focal as playwright-base
+
+
+
 
 
 
 FROM playwright-base as output-playwright
+# TODO: remove awscli dependency (157 MB extra)
+RUN apt-get update && apt-get install -y zip awscli && apt-get purge
 
 ARG APP_HOME
 ARG APP_DIST_HOME
 
 WORKDIR ${APP_DIST_HOME}
-ENV APP_DIST_HOME=${APP_DIST_HOME}
```

Comment:
are you sure `APP_DIST_HOME` as ENV can be scrapped for ARG? If I remember correctly, it's used as env at runtime.

Reply:
:eyes:

Reply:
Doesn't seem like it :shrug:

Reply:
I.e. I tried running without this set in my env when running the image locally without any error